README.md

```diff
@@ -201,3 +201,58 @@ AWS OpsWorks Chef Deployment
   ```ruby
   include_recipe 'datadog::dd-agent'
   ```
+
+Resources
+=========
+
+datadog_monitor
+---------------
+
+The `datadog_monitor` resource will help you to enable Agent integrations.
+
+The default action `:add` of this resource will automatically enable the integration by filling values in the configuration template file and set the correct permissions after having written the configuration on the filesystem. Also, it will take care of restarting the Agent.
+
+The `:remove` action disables an integration from a node.
+
+
+### Syntax
+
+```ruby
+datadog_monitor 'name' do
+  init_config                       Hash, default: {}
```

Comment:
nit: the chef official docs use a comment to mention the default, see for example: https://docs.chef.io/resource_directory.html